modules/nw-networkpolicy-delete.adoc

```diff
@@ -1,25 +1,41 @@
 // Module included in the following assemblies:
 //
 // * networking/network_policy/deleting-network-policy.adoc
-// * networking/configuring-networkpolicy.adoc
 // * post_installation_configuration/network-configuration.adoc
 
 [id="nw-networkpolicy-delete_{context}"]
-
 = Deleting a network policy
 
-You can delete a network policy.
+You can delete a network policy in a namespace.
+
+[NOTE]
+====
+If you log in with a user with the `cluster-admin` role then you can delete any network policy in the cluster.
+====
 
 .Prerequisites
 
 * You installed the OpenShift CLI (`oc`).
-* You are logged in to the cluster with a user with `cluster-admin` privileges.
+* You are logged in to the cluster with a user with `admin` privileges.
+* You are working in the namespace where the network policy exists.
 
 .Procedure
 
-* To delete a `NetworkPolicy` object, enter the following command. Replace `<policy-name>` with the name of the object.
+* To delete a `NetworkPolicy` object, enter the following command:
 +
 [source,terminal]
 ----
-$ oc delete networkpolicy <policy-name>
+$ oc delete networkpolicy <policy_name>
+----
++
+--
+where:
+
+`<policy_name>`:: Specifies the name of the network policy.
+--
++
+.Example output
+[source,text]
+----
+networkpolicy.networking.k8s.io/allow-same-namespace deleted
```

Comment:
Change to "multinetworkpolicy.k8s.cni.cncf.io "ingress-allow-diff-podselector-with-same-namespaceselector" deleted
"

Reply:
@weliang1, would this be correct for creating a multi-network policy?

`multinetworkpolicy.k8s.cni.cncf.io/default-deny created`